x-pack/plugins/monitoring/server/telemetry_collection/get_cluster_uuids.ts

```diff
@@ -28,10 +28,13 @@ export const getClusterUuids: ClusterDetailsGetter<CustomContext> = async (
 /**
  * Fetch the aggregated Cluster UUIDs from the monitoring cluster.
  */
-export function fetchClusterUuids(
-  { callCluster, start, end }: StatsCollectionConfig,
+export async function fetchClusterUuids(
+  { callCluster, timestamp }: StatsCollectionConfig,
   maxBucketSize: number
 ) {
+  const start = moment(timestamp).subtract(3, 'hours').toISOString();
```

Comment:
How did we get to 3 hours? We should document that somewhere and also define a constant for it:
```
MONITORING_COLLECTION_FETCH_INTERVAL = 3
```
or something like that.

It might also be worth our while trying to standardize how and where we add similar intervals, e.g. in the telemetry `FetcherTask` we also define the intervals [in-line](https://github.com/elastic/kibana/blob/master/src/plugins/telemetry/server/fetcher.ts#L59-L60) in the code, while in the same plugin, public side, we define the `REPORT_INTERVAL_MS` in the [common/constants.ts](https://github.com/elastic/kibana/blob/master/src/plugins/telemetry/common/constants.ts#L42) file.